styles/style.css

```diff
@@ -6,3 +6,127 @@ Green: #00B172
 White: #FFF
 
 */
+
+* {
+  font-family: sans-serif;
+}
+
+nav {
+  position: fixed;
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
```

Comment:
you could also add `width: 100%` so you would remove the right space on the navbar after adding position:fixed.